composer.json

```diff
@@ -4,34 +4,29 @@
   "homepage": "https://wordpress.org/plugins/stream/",
   "type": "wordpress-plugin",
   "license": "GPL-2.0-or-later",
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/WordPress/wordpress-develop.git"
```

Comment:
We use https://github.com/johnpbloch/wordpress now which doesn't require any build steps and is much faster to install, and can be used in combination of `wp-phpunit/wp-phpunit` to run the unit tests with any version of WP core.